src/Http/Owin/src/OwinEnvironment.cs

```diff
@@ -28,11 +28,18 @@ namespace Microsoft.AspNetCore.Owin
             Task<WebSocket>
         >;
 
+    /// <summary>
+    /// A strongly-typed wrapper of an OWIN environment.
```

Comment:
It's actually the other way around.
```suggestion
    /// A loosely-typed OWIN environment wrapper over an <see cref="HttpContext"/>.
```